docs/docs/jsx-in-depth.md

```diff
@@ -228,6 +236,18 @@ If you pass no value for a prop, it defaults to `true`. These two JSX expression
 
 In general, we don't recommend using this because it can be confused with the [ES6 object shorthand](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015) `{foo}` which is short for `{foo: foo}` rather than `{foo: true}`. This behavior is just there so that it matches the behavior of HTML.
 
+### Props with 'False' values
+
+If you don't pass a prop into a component, it will default to `null`, which is a falsey value in JavaScript.
+
+These two JSX expressions are equivalent.
+
+```js
+<MyTextBox />
+
+<MyTextBox autocomplete={false} />
```

Comment:
You are absolutely right, the prop is `undefined`. 

What I was trying to say is if you don't pass in `autocomplete`, you can do this inside the component 

if(autocomplete) {
  // if autocomplete is passed in as 'true' or other values
} else {
  // if autocomplete is not passed in or passed in as false
}

I already made these changes and recommitted.